test/apps/iodemo/io_demo.cc

```diff
@@ -1569,8 +1610,16 @@ class DemoClient : public P2pDemoCommon {
 
             elapsed_time = curr_time - start_time;
             if (elapsed_time > _test_opts.client_timeout) {
-                LOG << "timeout waiting for " << (_num_sent - _num_completed)
-                    << " replies";
+                std::stringstream ss, list_uncomp_ss;
```

Comment:
`ss` seems redundant, can use `UcxLog log` instead

Reply:
`list_uncomp_ss` can be removed if print counters in the end like:
```
1. conn1 info
2. conn2 info
N. connN info
total: X, to disconnect Y etc
```

Reply:
done

Reply:
> `ss` seems redundant, can use `UcxLog log` instead

not addressed